hail/src/main/scala/is/hail/annotations/StagedRegionValueBuilder.scala

```diff
@@ -77,6 +85,7 @@ class StagedRegionValueBuilder private (val mb: EmitMethodBuilder[_], val typ: P
     case t: PArray =>
       elementsOffset = mb.genFieldThisRef[Long]("srvb_array_addr")
       idx = mb.genFieldThisRef[Int]("srvb_array_idx")
+    case t: PNDArray => elementsOffset - mb.genFieldThisRef[Long]("srvb_ndarray_struct_addr")
```

Comment:
This should be an equal sign, not a minus sign. Not the source of my problem though.